Client/Frontend/Strings.swift

```diff
@@ -842,156 +842,198 @@ extension String {
-        public static let LimitedEditionWallpaper = MZLocalizedString(
-            key: "Onboarding.Wallpaper.Accessibility.LimitedEdition.v106",
-            tableName: nil,
-            value: "Limited Edition Wallpaper",
-            comment: "Accessibility label for the wallpaper onboarding modal displayed on top of the homepage. This describes to the user that which type of wallpaper they are seeing.")
+
+        public struct Intro {
+            public static let DescriptionPart1 = MZLocalizedString(
+                key: "Onboarding.IntroDescriptionPart1.v114",
+                tableName: "Onboarding",
+                value: "Indie. Non-profit. For good.",
+                comment: "String used to describes what Firefox is on the first onboarding page in our Onboarding screens. Indie means small independant.")
+            public static let DescriptionPart2 = MZLocalizedString(
+                key: "Onboarding.IntroDescriptionPart2.v114",
+                tableName: "Onboarding",
+                value: "Committed to the promise of a better Internet for everyone.",
+                comment: "String used to describes what Firefox is on the first onboarding page in our Onboarding screens.")
+            public static let Action = MZLocalizedString(
+                key: "Onboarding.IntroAction.v114",
+                tableName: "Onboarding",
+                value: "Get Started",
+                comment: "Describes the action on the first onboarding page in our Onboarding screen. This string will be on a button so user can continue the onboarding.")
+            public static let SecondaryAction = MZLocalizedString(
+                key: "Onboarding.IntroSecondaryAction.v114",
+                tableName: "Onboarding",
+                value: "Skip",
+                comment: "Describes the action on the first onboarding page in our Onboarding screen. This string will be on a button so user can skip the onboarding.")
+        }
+
+        public struct Wallpaper {
+            public static let Title = MZLocalizedString(
+                key: "Onboarding.WallpaperTitle.v114",
+                tableName: "Onboarding",
+                value: "Choose a %@ Wallpaper",
+                comment: "Title for the wallpaper onboarding page in our Onboarding screens. This describes to the user that they can choose different wallpapers. Placeholder is for app name.")
+            public static let Action = MZLocalizedString(
+                key: "Onboarding.WallpaperAction.v114",
+                tableName: "Onboarding",
+                value: "Set Wallpaper",
+                comment: "Description for the wallpaper onboarding page in our Onboarding screens. This describes to the user that they can set a wallpaper.")
+            public static let SelectorTitle = MZLocalizedString(
+                key: "Onboarding.Wallpaper.Title.v114",
```

Comment:
then this would need a slight change to avoid using the same key
```suggestion
                key: "Onboarding.Wallpaper.SelectorTitle.v114",
```